pkg/imagestore/imagestore.go

```diff
@@ -109,9 +111,24 @@ func validateVersions(versions []map[string]string) error {
 }
 
 func downloadURLToFile(url string, path string) error {
-	resp, err := http.Get(url)
+	var resp *http.Response
+	var err error
+
+	resp, err = http.Get(url)
```

Comment:
Isn't there just a flag not to verify tls?

Reply:
I'm pretty sure we do it in the agent alread'

Reply:
Yeah, that's what the block below is doing.
@CrystalChun we can probably just do the one call rather than waiting for an error and then doing the unverified one.

Also it might be better to create this client in `NewImageStore` and use it here rather than recreating it for each request.

Reply:
Yep, the flag in the [agent](https://github.com/openshift/assisted-installer-agent/blob/42d49b73098ae12d1879768f01c390fa952d1cae/src/session/inventory_session.go#L74) is set the same way. 

Good idea, I've moved the client creation to the `NewImageStore` function.